build.sbt

```
@@ -96,6 +96,24 @@ lazy val cbor = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     ),
   )
 
+lazy val json = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("json"))
+  .settings(
+    name := "http4s-fs2-data-json",
+    description := "Provides JSON codecs for http4s via fs2-data",
+    startYear := Some(2023),
+    tlVersionIntroduced := Map("2.12" -> "0.2", "2.13" -> "0.2", "3" -> "0.2"),
```

```suggestion
    startYear := Some(2024),
    tlVersionIntroduced := Map("2.12" -> "0.3", "2.13" -> "0.3", "3" -> "0.3"),
```

Let's update the year and set the correct version (will release this PR as 0.3 once merged). Run `sbt headersGenerate` as it might change the file headers.

---

OK, thank you for chiming in! I hope I understood them correctly in [96bccaa](https://github.com/http4s/http4s-fs2-data/pull/140/commits/96bccaa6e7d30b90f91fee360295e694ddfce18d).

---

Yes, exactly right!